### PR TITLE
change schedule view of admin

### PR DESCRIPTION
**Pull request by tuananhnguyen1708**

### CODE REVIEW

**Comment by tuananhnguyen1708** [comment]
screenshot from 2018-06-20 13-08-33

File: app/assets/javascripts/custom.js [hunk 1]
```diff
@@ -137,7 +137,9 @@ function printPrice(seats, scheduleId) {
 
 function printTicket(seats, scheduleId) {
   $('.booked-seat').data('schedule', scheduleId).empty();
-  $('.booked-seat').data('schedule', scheduleId).append(seats.join());
+  for(var i=0; i<seats.length ; i++){
```
Comment (DuongNT1512) [inline app/assets/javascripts/custom.js]: convention

File: app/views/admin/schedules/index.html.erb [hunk 2]
```diff
@@ -2,26 +2,29 @@
...
       </h1>
     </div>
   </div>
 
   <div class="row">
     <div class="col-lg-12">
-      <div>
-        <%= t "admin.schedule.filter"%>
+      <div class="filter-part">
+        <!-- <%= t "admin.schedule.filter"%> -->
```
Comment (DuongNT1512) [inline app/views/admin/schedules/index.html.erb]: k dùng thì bỏ hẳn đi nhé

File: app/views/admin/schedules/show.html.erb [hunk 3]
```diff
@@ -1,122 +1,159 @@
...
+            </div>
+          </div>
+          <div class="row line-info">
+            <div class="col-md-6 label-schedule-show">
+              <label><%= t "admin.schedule.table.stts" %>:</label>
+            </div>
+            <div class="col-md-6">
+              <div class="">
+                <%= @schedule.date < Date.current ? t("admin.schedule.status.done") :
+                status_schedule(@schedule) %>
```
Comment (DuongNT1512) [inline app/views/admin/schedules/show.html.erb]: indent

File: app/views/admin/schedules/show.html.erb [hunk 4]
```diff
@@ -1,122 +1,159 @@
...
+            <th scope="col"><%= t "admin.booked.type" %></th>
+            <th scope="col"><%= t "admin.booked.total_price" %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @schedule.bills.each_with_index do |bill, index| %>
+          <tr class="active">
+            <th scope="row"><%= index + 1 %></th>
+            <td><%= l bill.created_at.to_date,
+            format: :date_month_year_concise %></td>
```
Comment (DuongNT1512) [inline app/views/admin/schedules/show.html.erb]: indent

File: app/views/admin/schedules/show.html.erb [hunk 5]
```diff
@@ -1,122 +1,159 @@
...
+            <td><%= bill.user.phone_number %></td>
+            <td><%= bill.total_ticket %></td>
+            <td></td><td></td><td></td>
+            <td><%= bill.total_price %></td>
+          </tr>
+          <% bill.booked_seats.each do |booked| %>
+          <tr>
```
Comment (DuongNT1512) [inline app/views/admin/schedules/show.html.erb]: indent

**Pull request merged**